client/transactions/declarations.d.ts

```diff
@@ -0,0 +1,48 @@
+declare module '@woocommerce/explat' {
+	type ExperimentProps = {
+		name: string;
+		defaultExperience: JSX.Element;
+		treatmentExperience?: JSX.Element;
+		loadingExperience?: JSX.Element;
+	};
+
+	const Experiment: ( props: ExperimentProps ) => JSX.Element;
+}
+
+declare module 'interpolate-components' {
```

Comment:
I still seem to be getting some issues with the definition when trying to use this inside the `autocomplete.tsx` file which I need to investigate!

Reply:
Yeah, when declaring a default export you have to use `export = <defaultExport>`, in this case that would be:

```ts
declare module 'interpolate-components' {
    // ...
    export = interpolateComponents;
}
```

Reply:
I fixed this in d6ed25bee2bb17ae80cf9817c2164eaa693badec 🙂

Reply:
Thanks! That's good to know, will keep that in mind for next time :)